test/controllers/EditSetControllerSpec.scala

```diff
@@ -306,6 +307,31 @@ class EditSetControllerSpec extends BaseSpec {
 
       status(editRecordPage) mustBe SEE_OTHER
     }
+
+    "respond with BAD_REQUEST when form has errors, preserving the CCR" in {
+      val ccrToAssert = "I'm looking for this CCR specifically"
```

Comment:
Perhaps we could use something more realistic, like `COAL 80/80/1`?

Otherwise, someone might get the impression that it was that CCR that caused the error.

Reply:
Agreed; it's better to use realistic data, chaged.

Reply:
Confirmed.